keep/providers/providers_service.py

```diff
@@ -415,4 +415,8 @@ def provision_providers_from_env(tenant_id: str):
     def get_provider_logs(
         tenant_id: str, provider_id: str
     ) -> List[ProviderExecutionLog]:
+        import os
+        if not os.environ.get("KEEP_STORE_PROVIDER_LOGS", "false").lower() == "true":
```

Comment:
import this up:
`from keep.api.core.config import config`
and use
```suggestion
        if not config("KEEP_STORE_PROVIDER_LOGS", cast=bool):
```